specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -186,6 +186,137 @@
           }
         }
       }
+    },
+    "/analyze-conversation/jobs": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "description": "Submit a long conversation for analysis. Specify one or more unique tasks to be executed as a long-running operation.",
+        "operationId": "AnalyzeConversation_SubmitJob",
+        "summary": "Submit analysis job for conversations",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "description": "The conversations to analyze and one or more tasks to execute on the conversations.",
```

Comment:
nit: remove `on the conversations.` suffix

Reply:
yes.